Installation/Mapping/oc.githubPubliccodeComponent.mapping.json

```diff
@@ -2,7 +2,7 @@
   "title": "GithubPubliccodeToComponent",
   "$id": "https://api.github.com/oc.githubPubliccodeComponent.mapping.json",
   "$schema": "https://docs.commongateway.nl/schemas/Mapping.schema.json",
-  "version": "0.0.1",
+  "version": "0.0.3",
```

Comment:
You are skipping 0.0.2 (but that isn't very important)

Reply:
In te pr from Robert the version was set to 2